Server/Content/Sources/App/Models/Text.swift

```diff
@@ -0,0 +1,27 @@
+import Fluent
+import Vapor
+
+final class Text: Model, Content {
+    static let schema = "texts"
+    
+    @ID(key: .id)
+    var id: UUID?
+
+    @Field(key: "title")
+    var title: String
+    
+    @Field(key: "text")
+    var text: String
+    
+    @Field(key: "backward_text")
+    var backwardText: String?
+
+    init() { }
```

Comment:
Is this init is necessary?

Reply:
Yes. Thrown buildtime error if without init: `Type 'Article' does not conform to protocol 'Fields'`